railties/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Remove `config` options from `default_options`
```

Comment:
This change don't need a CHANGELOG since it will not affect the users. Could you remove it?